<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ComoxBox_demo</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>4</y>
     <width>881</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>简单ComoBox</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QPushButton" name="btnClearList">
         <property name="text">
          <string>清空列表</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QComboBox" name="comdItems">
         <item>
          <property name="text">
           <string>宜宾</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>成都</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>乐山</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>自贡</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>泸州</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>可编辑</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btnInitItems">
         <property name="text">
          <string>初始化列表</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>用户数据ComboBox</string>
      </property>
      <widget class="QPushButton" name="btnInitCities">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>321</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>初始化城市+区号</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comdCitys">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>321</width>
         <height>26</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>选择一个城市</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>显示信息</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnClear">
         <property name="text">
          <string>清空文本</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
